plugins/package-managers/bazel/src/funTest/assets/projects/synthetic/bazel/.bazelversion

```diff
@@ -0,0 +1 @@
+7.0.1
```

Comment:
Nits:
- Add a final newline.
- Do not start a paragraph for the last sentence in the commit message.